test/unit/logic/block.js

```diff
@@ -362,143 +396,187 @@ describe('block', () => {
-							data.transactions.length
-						);
-						expect(
-							expectedOrderOfTransactions(generatedBlock.transactions)
-						).to.equal(true);
-						return expect(transactionsOrder).to.eql(correctOrderOfTransactions);
+				beforeEach(done => {
+					multipleMultisigTx = Array(...Array(5)).map(() => {
+						return transactionsByTypes[transactionTypes.MULTI];
 					});
+					data.transactions = transactions.concat(multipleMultisigTx);
+					generatedBlock = block.create(data);
+					transactionsOrder = generatedBlock.transactions.map(trs => {
+						return trs.type;
+					});
+					done();
 				});
 
-				describe('shuffled', () => {
-					var multipleMultisigTx;
-					var generatedBlock;
-					var transactionsOrder;
-
-					beforeEach(done => {
-						multipleMultisigTx = Array(...Array(5)).map(() => {
-							return transactionsByTypes[transactionTypes.MULTI];
-						});
-						data.transactions = _.shuffle(
-							transactions.concat(multipleMultisigTx)
-						);
-						generatedBlock = block.create(data);
-						transactionsOrder = generatedBlock.transactions.map(trs => {
-							return trs.type;
-						});
-						done();
-					});
+				it('should sort transactions in the correct order', () => {
+					expect(generatedBlock.transactions.length).to.equal(
+						data.transactions.length
+					);
+					expect(
+						expectedOrderOfTransactions(generatedBlock.transactions)
+					).to.equal(true);
+					return expect(transactionsOrder).to.eql(correctOrderOfTransactions);
+				});
+			});
+
+			describe('shuffled', () => {
+				var multipleMultisigTx;
+				var generatedBlock;
+				var transactionsOrder;
 
-					it('should sort transactions in the correct order', () => {
-						expect(generatedBlock.transactions.length).to.equal(
-							data.transactions.length
-						);
-						expect(
-							expectedOrderOfTransactions(generatedBlock.transactions)
-						).to.equal(true);
-						return expect(transactionsOrder).to.eql(correctOrderOfTransactions);
+				beforeEach(done => {
+					multipleMultisigTx = Array(...Array(5)).map(() => {
+						return transactionsByTypes[transactionTypes.MULTI];
+					});
+					data.transactions = _.shuffle(
+						transactions.concat(multipleMultisigTx)
+					);
+					generatedBlock = block.create(data);
+					transactionsOrder = generatedBlock.transactions.map(trs => {
+						return trs.type;
 					});
+					done();
+				});
+
+				it('should sort transactions in the correct order', () => {
+					expect(generatedBlock.transactions.length).to.equal(
+						data.transactions.length
+					);
+					expect(
+						expectedOrderOfTransactions(generatedBlock.transactions)
+					).to.equal(true);
+					return expect(transactionsOrder).to.eql(correctOrderOfTransactions);
 				});
 			});
 		});
 	});
+
+	describe('sign', () => {
+		it('should return a signed block using keypair', () => {
+			return expect(block.sign(blockData, validKeypair)).to.be.string;
+		});
+	});
+
+	describe('getBytes', () => {
```

Comment:
add at least one hardcoded result that we will compare against

Reply:
added to test to compare the bytes against different scenarios.